src/box/box.h

```diff
@@ -620,6 +649,23 @@ box_iproto_send(uint64_t sid,
 		const char *header, const char *header_end,
 		const char *body, const char *body_end);
 
+/**
+ * Sets an IPROTO request handler with the provided context for the
+ * given request type.
+ *
+ * \param request_type request type code from iproto_type enumeration
+ *                     (src/box/iproto_constants.h);
+ *                     use IPROTO_UNKNOWN for overriding the handler of unknown
+ *                     request types;
+ * \param handler IPROTO request handler; passing NULL resets the corresponding
+ *                handler
+ * \param ctx context passed to handler
+ * \retval -1 on error (check box_error_last())
+ * \retval 0 on success
```

Comment:
AFAIU the function may yield (to configure IPROTO threads). We should mention it in the comment.

Reply:
Added a NB comment.